server/app/jobs/node_cleanup_job.rb

```diff
@@ -25,7 +25,8 @@ def perform
   end
 
   def cleanup_stale_connections
-    HostNode.where(:last_seen_at.lt => NODE_DISCONNECT_TIMEOUT.ago).each do |node|
+    HostNode.where(:connected => true, :last_seen_at.lt => NODE_DISCONNECT_TIMEOUT.ago).each do |node|
```

Comment:
After it sets the `HostNode.connected = false`, it shouldn't re-log the `Disconnecting stale node ...` warning anymore for each `HostNode`.

The reason for this change is to avoid re-logging that new warning every minute for each disconnected node.